ipaserver/install/certs.py

```diff
@@ -501,12 +501,17 @@ def create_password_conf(self):
         This is the format of mod_nss pin files.
         """
         ipautil.backup_file(self.pwd_conf)
-        f = open(self.pwd_conf, "w")
-        f.write("internal:")
-        pwdfile = open(self.passwd_fname)
-        f.write(pwdfile.read())
-        f.close()
-        pwdfile.close()
+
+        with open(self.passwd_fname) as pwdfile:
+            password = pwdfile.read()
+
+        with open(self.pwd_conf, "w") as f:
+            f.write("internal:")
+            f.write(password)
+            f.write("\nNSS FIPS 140-2 Certificate DB:")
+            f.write(password)
```

Comment:
I did not run into any issues with it not being there. It was not there when there was only the `internal` token so I assume there's no need for it here as well.